app/services/policies/policies_list_service.rb

```diff
@@ -6,11 +6,16 @@
 class Policies::PoliciesListService
   def self.call(account, builtin: true)
     list = PolicyList.new
-    if builtin
-      response = ::JSONClient.get(ThreeScale.config.sandbox_proxy.apicast_registry_url)
-      return unless response.ok?
-
-      list.merge!(PolicyList.from_hash(response.body['policies']))
+    apicast_registry_url = ThreeScale.config.sandbox_proxy.apicast_registry_url
+    if builtin && apicast_registry_url
+      begin
+        response = ::JSONClient.get(apicast_registry_url)
+        return unless response.ok?
+        list.merge!(PolicyList.from_hash(response.body['policies']))
+      rescue => error
+        Rails.logger.error("APIcast could not be found at #{apicast_registry_url}. Error: #{error}")
+        # TODO: notify UI that something went wrong
```

Comment:
I see this is [THREESCALE-2312](https://issues.jboss.org/browse/THREESCALE-2312), ok 👍